src/webgl/interaction.js

```diff
@@ -29,6 +29,9 @@ import * as constants from '../core/constants';
  * Setting this to true makes mobile interactions smoother by preventing
  * accidental interactions with the page while orbiting. But if you're already
  * doing it via css or want the default touch actions, consider setting it to false.
+ * free - Boolean, default value is false.
+ * Setting this to true will always rotate in the direction you move the mouse or touch pointer.
```

Comment:
Maybe we should mention that the default behavior rotates about the x and y axes so people know what they're comparing against

Reply:
Sure, I should explain the difference from the default case better...
How about this?
"By default, horizontal movement of the mouse or touch pointer rotates the camera around the y-axis, and vertical movement rotates the camera around the x-axis. But if setting this option to true, the camera always rotates in the direction the pointer is moving. Regarding zooming and moving, the behavior is the same regardless of true/false."

I also thought that "freeRotation" might be more appropriate than "free" since this option only affects rotation...